File: apps/myjobhunter/backend/app/schemas/profile/education.py
"""Education schemas — Phase 1 stub."""
import uuid
from datetime import datetime

from pydantic import BaseModel


class EducationRead(BaseModel):
    id: uuid.UUID
    profile_id: uuid.UUID
    school: str
    degree: str | None
    field: str | None
    start_year: int | None
    end_year: int | None
    created_at: datetime

    model_config = {"from_attributes": True}

File: apps/myjobhunter/frontend/src/types/education/education.ts
/**
 * Education as returned by the backend.
 * Mirrors EducationResponse in backend/app/schemas/profile/education_response.py.
 */
export interface Education {
  id: string;
  user_id: string;
  profile_id: string;
  school: string;
  degree: string | null;
  field: string | null;
  start_year: number | null;
  end_year: number | null;
  gpa: string | null;
  created_at: string;
  updated_at: string;
}
